internal/models/access_flow.go

```diff
@@ -3,20 +3,22 @@ package models
 import (
 	"github.com/hashicorp/terraform-plugin-framework/attr"
 	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
 )
 
 // AccessFlowModel describes the resource data model.
 type AccessFlowModel struct {
-	ID                 types.String        `tfsdk:"id"`
-	Name               types.String        `tfsdk:"name"`
-	Active             types.Bool          `tfsdk:"active"`
-	RevokeAfterInSec   types.Number        `tfsdk:"revoke_after_in_sec"`
-	Trigger            *Trigger            `tfsdk:"trigger"`
-	Grantees           types.Set           `tfsdk:"grantees"`
-	IntegrationTargets []IntegrationTarget `tfsdk:"integration_targets"`
-	BundleTargets      []BundleTarget      `tfsdk:"bundle_targets"`
-	Approvers          types.Set           `tfsdk:"approvers"`
-	Settings           *Settings           `tfsdk:"settings"`
+	ID                  types.String        `tfsdk:"id"`
+	Name                types.String        `tfsdk:"name"`
+	Active              types.Bool          `tfsdk:"active"`
+	RevokeAfterInSec    types.Number        `tfsdk:"revoke_after_in_sec"`
+	Trigger             *Trigger            `tfsdk:"trigger"`
+	Grantees            types.Set           `tfsdk:"grantees"`
+	GranteesFilterGroup types.Object        `tfsdk:"grantees_filter_group"`
+	IntegrationTargets  []IntegrationTarget `tfsdk:"integration_targets"`
```

Comment:
why is that types.Object and not GranteeFilterGroup or so?

Reply:
lets talk about it